src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -0,0 +1,198 @@
+/**
+ *  View control for the sidebar directions tab
+ *
+ */
+CAC.Control.SidebarDirections = (function ($, MapTemplates, Routing, UserPreferences) {
+
+    'use strict';
+
+    var defaults = {};
+    var options = {};
+
+    var currentItinerary = null;
+    var datepicker = null;
+
+    var directions = {
+        origin: null,
+        destination: null
+    };
+
+    var mapControl = null;
+    var tabControl = null;
+
+    function SidebarDirectionsControl(params) {
+        options = $.extend({}, defaults, params);
+        mapControl = options.mapControl;
+        tabControl = options.tabControl;
+
+        // Plan a trip using information provided
+        $('section.directions button[type=submit]').click($.proxy(planTrip, this));
+
+        // initiallize date/time picker
+        datepicker = $('#datetimeDirections').datetimepicker({useCurrent: true});
+
+        setFromUserPreferences();
+    }
+
+    SidebarDirectionsControl.prototype = {
+        setDestination: setDestination,
+        setDirections: setDirections
+    };
+
+    return SidebarDirectionsControl;
+
+    function planTrip() {
+        if (!(directions.origin && directions.destination)) {
+            setDirectionsError();
+            return;
+        }
+
+        var picker = $('#datetimeDirections').data('DateTimePicker');
```

Comment:
For all of these css selector ids/classes, I think it would be better to use the jQuery plugin pattern of having them specified as defaults and then allowing them to be overridden with options, if desired, when the controller is initialized. Moving all of them outside the controller logic would make things cleaner.